site/content/api/Command-And-Control-API.md

```diff
@@ -40,33 +39,24 @@ The following table provides an overview of the properties the *Business Applica
 | :--- | :-------: | :------- | :-------- | :---------- |
 | *message-id* | yes | *properties* | *string* | MUST contain the message ID used to match incoming control message with the response message containing result of execution. |
 | *subject* | yes | *properties* | *string* | MUST contain the command to be executed by a device. |
-| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. |
+| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the sender should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
+| *ttl* | no | *properties* | *int* | Duration in milliseconds for which the message is to be considered 'live'. Messages that live longer than their expiration time will be discarded (or dead lettered). |
```

Comment:
I still don't understand the purpose of this property. Does this indicate that Hono is supposed to *buffer* the request for that amount of time? Is this the minimum (?) time that Hono should wait for a reply from the device until it returns an error to the application?

Reply:
I introduced this at the time I defined the API between Business Application and the Adapter to show, that this standard AMQP 1.0 properties are there and could be used. Now it is between Business Application and a device - without a concrete idea of all device specific implementations of this side of the communication. So it might be better to not list them here since I am not sure at the moment if they will be forwarded to the device or only interpreted by the adapter. So I would remove ttl and content-type from the API definition. WDYT?

Reply:
Unless we really know what they are needed for, yes, I agree, we should remove them for the time being.